rakelib/test.rake

```diff
@@ -1,33 +1,29 @@
+def run_rspec(*args)
+  require "logstash/environment"
+  LogStash::Environment.set_gem_paths!
+  require "rspec/core/runner"
+  require "rspec"
+  RSpec::Core::Runner.run([*args])
+end
```

Comment:
I do not recall the actual problem, but I am sure this line wasn't working on windows when we were trying to run the test suite on all the plugins, we had to hack our ways to it. Do you remember @jordansissel ?